README.md

```diff
@@ -22,23 +22,27 @@ Build and bring up the containers.
 
     $ make up
 
-Create an admin user. You'll be prompted for a password.
+You may seed the inventory with a default user, organization, and dataset by running the following command in the folder while the docker-compose is still up and has finished running:
 
-    $ docker-compose run --rm app paster --plugin=ckan sysadmin add admin -c /etc/ckan/production.ini
+    $ docker container exec -it inventoryapp_app_1 /opt/inventory-app/seed.sh
```

Comment:
This should be docker-compose, which will figure out the container name for you and defaults to an interactive shell. I also prefer `run --rm`, so you don't have to have a running container, but depending on how the command works, this `run` isn't always an option.

```suggestion
    $ docker-compose exec app /opt/inventory-app/seed.sh
```